templates/explore/repo_list.tmpl

```diff
@@ -54,9 +54,9 @@
 				{{$description := .DescriptionHTML $.Context}}
 				{{if $description}}<p>{{$description}}</p>{{end}}
 				{{if .Topics}}
-					<div class="ui tags">
+					<div class="gt-df gt-fw gt-gap-2 gt-mb-3 gt-font-normal gt-font-14">
```

Comment:
Normal is weight, 14 is size, so two classes is correct to override both. It might be that fomantic styles are interfering too much, not sure if there are better solutions.

Reply:
Then do not use "ui basic label", overwriting styles is really fragile, and it bloats the code.

Reply:
ok